x-pack/plugins/uptime/server/kibana.index.ts

```diff
@@ -39,18 +39,14 @@ export const initServerWithKibana = (server: UptimeCoreSetup, plugins: UptimeCor
       all: {
         app: ['uptime', 'kibana'],
         catalogue: ['uptime'],
-        api: [
-          'uptime-read',
-          'uptime-write',
-          'actions-read',
-          'actions-all',
-          'alerting-read',
-          'alerting-all',
-        ],
+        api: ['uptime-read', 'uptime-write', 'actions-read', 'actions-all'],
         savedObject: {
           all: [umDynamicSettings.name, 'alert', 'action', 'action_task_params'],
           read: [],
         },
+        alerting: {
+          all: ['xpack.uptime.alerts.tls', 'xpack.uptime.alerts.monitorStatus'],
```

Comment:
Still wondering about access to the built-in alert types.  Will users with access to uptime alerting have access to the built-in alerts?  I'm not sure if that's appropriate or not, but it seems like we should probably "by default" allow users with any alerting privs to get access to the built-in ones - though of course a solution might want to restrict this as well.

I'm worried about cases where customers won't be able to find the built-in alerts easily, even though we will certainly be doc'ing them, building examples, and then the customer being confused by not seeing them anywhere.

I guess an alternative question is - when **WOULD** you specifically add built-in alerts here?

Reply:
The way it's modelled now all users have access to built-in alerts, but only in Alerts Management.
If Uptime want users to be able to create an Index Threshold alert inside of Uptime, then they'll need to explicitly grant users permission to create it from their own feature privileges setup.

This *doesn't* control whether it's displayed, only whether they're allowed to create it with uptime as the consumer via the API.
The client side decision as to whether they display it is different and at the moment it's set to only display the solutions alerts inside of their flyouts (this is part of 7.9).
We said we'd reconsider this later in the sync last week.